apps/freelance/src/pages/FindJobs/index.tsx

```diff
@@ -36,25 +43,36 @@ const FindJobs = () => {
         />
       </PageBar>
 
-      <List
-        dataSource={jobsList}
-        renderItem={item => (
-          <List.Item>
-            <JobCard
-              key={item.id}
-              title={item.title}
-              description={item.description}
-              category={item.category}
-              date={item.date}
-              duration={item.duration}
-              owner={`${item.owner.first_name} ${item.owner.last_name}`}
-              rate={item.rate}
-            />
-          </List.Item>
-        )}
-      />
-
-      <StyledPagination />
+      {isLoading ? (
+        <Spin size="large" />
+      ) : (
```

Comment:
loader better to move to the wrapper, you avoid ternary operator in each component

Reply:
@lyudad refactored with loading prop in List component